prov/efa/src/rdm/efa_rdm_ep_fiops.c

```diff
@@ -1288,6 +1289,60 @@ static int efa_rdm_ep_ctrl(struct fid *fid, int command, void *arg)
 	return ret;
 }
 
+static int efa_rdm_ep_cancel_match_recv(struct dlist_entry *item,
+				         const void *context)
+{
+	struct efa_rdm_ope *rxe = container_of(item, struct efa_rdm_ope, entry);
+	return rxe->cq_entry.op_context == context;
+}
+
+/**
+ * @brief Cancel a recv in ep->user_recv_rxe_list
+ *
+ * @param ep efa_rdm_ep
+ * @param context pointer to the context to be cancelled
+ * @return ssize_t 0 on success, negative integer on failure.
+ */
+static
+ssize_t efa_rdm_ep_cancel_zcpy_recv(struct efa_rdm_ep *ep,
+				     void *context)
+{
+	struct dlist_entry *entry;
+	struct efa_rdm_ope *rxe;
+	struct fi_cq_err_entry err_entry;
```

Comment:
Zero-initialize and avoid the `memset()` below.

```suggestion
	struct fi_cq_err_entry err_entry = {0};
```

Reply:
Updated